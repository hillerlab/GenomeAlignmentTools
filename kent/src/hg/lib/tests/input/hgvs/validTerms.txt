# A variant that is famously known as "MTHFR C677T" or "MTHFR 677C>T" -- but the 677 is incorrect (from an old sequence ~1995):
NM_005957.4:c.665C>T
# Same variant in LRG transcript, protein and genomic coords:
LRG_726t1:c.665C>T
LRG_726p1:p.Ala222Val
LRG_726:g.14783C>T

# These three are equivalent (although p. is less specific):
NM_003002.3:c.276C>A
NP_002993.1:p.Asp92Glu
NP_002993.1:p.D92E

# This is not HGVS but seems to be a commonly requested format and is easy to transform into the internal representation of HGVS:
ALK G1494E

# More gene symbol HGVS-ish
RERE:Thr758Serfs
GBA:p.Leu29Alafs*18
H6PD Val320=
MTHFR Pro101_Gly102insLeuTyrIleAspValThrTrpHisProAlaGlyAspPro
YARS p.Val153_Val156del
PLOD1 Glu532del
PPT1 p.Gln91_Cys96delinsPhe
PLOD1:p.532
# From a user via b0b:
NM_198056.2:pAla735Val
NM_198056.2:pArg1512Trp

NM_003098.2:c.287G>C
NM_007294.3:c.2231C>A
NM_007294.3(BRCA1):c.2231C>A
