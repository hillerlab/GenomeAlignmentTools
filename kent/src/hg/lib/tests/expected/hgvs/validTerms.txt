# db: hg38
# inputFile: input/hgvs/validTerms.txt
# A variant that is famously known as "MTHFR C677T" or "MTHFR 677C>T" -- but the 677 is incorrect (from an old sequence ~1995):
chr1	11796320	11796321	NM_005957.4:c.665C>T	0	-
# Same variant in LRG transcript, protein and genomic coords:
chr1	11796320	11796321	LRG_726t1:c.665C>T	0	-
chr1	11796319	11796322	LRG_726p1:p.Ala222Val	0	-
chr1	11796320	11796321	LRG_726:g.14783C>T	0	+

# These three are equivalent (although p. is less specific):
chr11	112088972	112088973	NM_003002.3:c.276C>A	0	+
chr11	112088970	112088973	NP_002993.1:p.Asp92Glu	0	+
chr11	112088970	112088973	NP_002993.1:p.D92E	0	+

# This is not HGVS but seems to be a commonly requested format and is easy to transform into the internal representation of HGVS:
chr2	29193604	29193607	ALK G1494E	0	-

# More gene symbol HGVS-ish
chr1	8361232	8361235	RERE:Thr758Serfs	0	-
chr1	155240657	155240660	GBA:p.Leu29Alafs*18	0	-
chr1	9262270	9262273	H6PD Val320=	0	+
chr1	11801329	11801335	MTHFR Pro101_Gly102insLeuTyrIleAspValThrTrpHisProAlaGlyAspPro	0	-
chr1	32806523	32806535	YARS p.Val153_Val156del	0	-
chr1	11966259	11966262	PLOD1 Glu532del	0	+
chr1	40092118	40092136	PPT1 p.Gln91_Cys96delinsPhe	0	-
chr1	11966259	11966262	PLOD1:p.532	0	+
# From a user via b0b:
chr3	38597785	38597788	NM_198056.2:pAla735Val	0	-
chr3	38555661	38555664	NM_198056.2:pArg1512Trp	0	-

chr20	33443333	33443334	NM_003098.2:c.287G>C	0	-
chr17	43093299	43093300	NM_007294.3:c.2231C>A	0	-
chr17	43093299	43093300	NM_007294.3(BRCA1):c.2231C>A	0	-
